ntbs-service/wwwroot/css/notification.scss

```diff
@@ -1,28 +1,28 @@
-
-.manage-notification-section {
-    width: 300px;
-    padding-top: 10px;
-    border-top: 10px solid #1d70b8;
-}
-  
-.notification-sideline-container {
-    width: 25%;
-    display: inline-block;
-    flex-wrap: nowrap;
-}
-
-.notification-body-container {
-    width: 50%;
-    display: inline-block;
-    flex-wrap: nowrap;
-}
-
-.notification-container {
-    display: flex;
-    flex-direction: row;
-    flex-wrap: nowrap;
-}
-
-.patient-layout-flexbox {
-    display: flex;
+
+.manage-notification-section {
+    width: 300px;
```

Comment:
could you please change this to "max-width" instead of width when committing?

Reply:
sure!